package.json

```diff
@@ -23,7 +23,7 @@
     "prepublish": "yarn clean && yarn build",
     "start": "start-storybook -p 6006",
     "deploy-storybook": "storybook-to-ghpages",
-    "cp-cloud": "cp -R ./dist ../publishe/cloud-frontend/node_modules/@netdata/charts/",
```

Comment:
?

Reply:
@burbuli8ra i accidentally added this in previous pr, here i'm just reverting it